pyproject.toml

```diff
@@ -60,7 +60,8 @@ lint.ignore = [
   'PLR2004',
   'PTH',
   'RET505',
-  'S',
+  'S101',    # Won't fix
+  'S311',    # Won't fix
```

Comment:
I am not finding any instances of this in the codebase. Any reason to ignore it just yet?